ods_ci/build/Dockerfile_interop

```diff
@@ -0,0 +1,54 @@
+FROM quay.io/centos/centos:stream9
+
+# Use this build arg to set any default test script arguments
+ENV RUN_SCRIPT_ARGS=${RUN_SCRIPT_ARGS}
+ENV ROBOT_EXTRA_ARGS=''
+ENV SET_ENVIRONMENT=0
+ENV RETURN_PW=0
+ENV OC_HOST=${OC_HOST}
+ENV RUN_FROM_CONTAINER=1
+ENV SE_BROWSER_PATH=/usr/bin/chromium-browser
+ARG OC_VERSION=4.13
+ARG OC_CHANNEL=stable
+ARG PYTHON_VERSION=3.11
+
+ENV ODS_VENV="/ods_venv" \
+    HOME="/ods_venv" \
+    PATH="ods_venv/bin:${PATH}" \
+    AM_I_IN_CONTAINER="Yes" \
+    PYTHONUNBUFFERED="True"
+
+WORKDIR /ods_venv
+
+COPY . ${ODS_VENV}
+COPY ods_ci/test-variables.yml.example ${ODS_VENV}/ods_ci/test-variables.yml
+
+RUN dnf install epel-release -y &&\
+    dnf -y update &&\
+    dnf install -y jq git unzip chromium chromedriver httpd-tools gcc \
+                   python3 python3-devel python3-distro python-pip python${PYTHON_VERSION} python${PYTHON_VERSION}-devel &&\
+    dnf clean all && rm -rf /var/cache/yum &&\
+    curl --proto "=https" -L https://github.com/mikefarah/yq/releases/download/v4.34.1/yq_linux_amd64 -o /usr/bin/yq &&\
+    chmod +x /usr/bin/yq &&\
+    curl --proto "=https" -L https://mirror.openshift.com/pub/openshift-v$(echo ${OC_VERSION} | cut -d'.' -f 1)/x86_64/clients/ocp/${OC_CHANNEL}-${OC_VERSION}/openshift-client-linux.tar.gz -o ${HOME}/oc_client.tar.gz && \
+    tar xvf ${HOME}/oc_client.tar.gz -C /usr/local/bin/ && \
+    rm -rf ${HOME}/oc_client.tar.gz && rm /usr/local/bin/README.md && chmod 755 /usr/local/bin/oc && oc version --client && \
+    curl --proto "=https" -L https://github.com/openshift-online/ocm-cli/releases/download/v0.1.62/ocm-linux-amd64 -o ${HOME}/ocm && \
+    mv ${HOME}/ocm /usr/local/bin/ && chmod 755 /usr/local/bin/ocm && ocm version
+
+RUN alternatives --install /usr/local/bin/python3 python3 /usr/bin/python${PYTHON_VERSION} 1
+RUN python3 --version
+RUN curl -sSL https://install.python-poetry.org | python3 -
+ENV PATH="${PATH}:${HOME}/.local/bin"
+RUN poetry install
+
+ENV REQUESTS_CA_BUNDLE="/etc/pki/tls/certs/ca-bundle.crt"
+RUN curl -L https://certs.corp.redhat.com/certs/Current-IT-Root-CAs.pem \
+         -o /etc/pki/ca-trust/source/anchors/Current-IT-Root-CAs.pem && \
+    update-ca-trust
+
+RUN chgrp -R 0 ${ODS_VENV} && \
+    chmod -R g+rwX ${ODS_VENV}
+
+USER 1001
```

Comment:
if this runs on OCP, it should assign a random user regardless USER statement afaik

Reply:
Correct, OCP will automatically reassign a random user ID, but it must be a non-root user; otherwise, we encounter a permission denied issue in OpenShift CI.